docs/sources/developers/http_api/correlations.md

```diff
@@ -0,0 +1,71 @@
+---
+aliases:
+  - /docs/grafana/latest/developers/http_api/correlations/
+  - /docs/grafana/latest/http_api/correlations/
+description: Grafana Correlations HTTP API
+keywords:
+  - grafana
+  - http
+  - documentation
+  - api
+  - correlations
+  - Glue
+title: 'Correlations HTTP API '
+---
+
+# Correlations API
+
+This API can be used to define correlations between data sources.
+
+## Create correlations
+
+`POST /api/datasources/:sourceUid/correlations`
+
+Creates a correlation between two data sources.
```

Comment:
"Creates a correlation between two data sources - the source datasource indicated by the path UID, and the target datasource which is specified in the body." 

Since we have datasource IDs in two different places, I would specify that in the docs in case someone misses the path and goes right for the payload. I'm also wondering if it would be worth specifying here if two of the same UIDs, swapped for source and target are equivalent to each other.

Reply:
the text looks way better. 👍 

regarding the second part i think this needs to be better explained in the docs (correlations are not bi-directional) on what this feature is about and what it does exactly. we should take a look at that soon @josiahg